front/src/applications/operationalStudies/components/SimulationResults/ChartHelpers/drawArea.jsx

```diff
@@ -1,28 +0,0 @@
-import * as d3 from 'd3';
-
-const drawArea = (chart, classes, dataSimulation, groupID, interpolation, keyValues, rotate) => {
```

Comment:
right now it’s only called with keyValues[0] === 'position' but I guess it’s meant to be a more generic function to draw areas with different axes. I’m not sure about completely removing keyValues.

At least document in the PR description that this is when  keyValues was removed so it’s easily found